aten/src/ATen/cuda/CublasHandlePool.cpp

```diff
@@ -41,31 +41,38 @@ void clearCublasWorkspaces() {
 
 size_t parseChosenWorkspaceSize() {
   const char * val = getenv("CUBLAS_WORKSPACE_CONFIG");
+  /* :4096:2:16:8 default, 32MiB for Hopper */
+  int device;
+  cudaDeviceProp* properties = at::cuda::getDeviceProperties(cudaGetDevice(&device));
```

Comment:
Hmm, `cudaGetDevce()` returns an error code isn't it? Which is 0 in case of success
```suggestion
  cudaDeviceProp* properties = getCurrentDeviceProperties();
```

Reply:
Good catch, thanks!